addons/turbowarp-player/userscript.js

```diff
@@ -26,7 +26,7 @@ export default async function ({ addon, console, msg }) {
     button.title = "TurboWarp";
   }
 
-  button.onclick = async () => {
```

Comment:
Maybe I'm dumb (and I know you didn't write this code), but shouldn't this be an event listener? I'm pretty sure the standard is to not use the old .ons.

Reply:
It's okay if the event is being registered into an element that was created by the same addon. The new userscript docs will make this explicit